src/ParsedCertificate.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+/*
+ * This file is part of the ACME PHP library.
+ *
+ * (c) Titouan Galopin <galopintitouan@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace AcmePhp\Ssl;
+
+use Webmozart\Assert\Assert;
+
+/**
+ * Represent the content of a parsed certificate.
+ *
+ * @author Jérémy Derussé <jeremy@derusse.com>
+ */
+class ParsedCertificate
```

Comment:
What do you think of  extending Certificate base classe? Seems logical at the first glimpse but I'm not sure